ansysFiles/talkToAnsys.py

```diff
@@ -2,14 +2,15 @@
 import subprocess
 import shutil
 
+fluent_path = shutil.which("fluent")
+
+WB_path=fluent_path[0:int(fluent_path.find("fluent"))] + r"Framework\bin\Win64\RunWB2.exe"
```

Comment:
This only works on Windows, you may consider adding content to work with windows or mac/linux

Reply:
Do you recommendations on how to make this Mac/Linux compatible?